Init documentation for the network operators. (#117)
LiqoTech/liqo: docs/design/liqonet_tunnelEndpoint.md

# TunnelEndpoint-Operator
## Overview
This component is in charge of establishing a VPN tunnel with the peering clusters, which is created between the (local) Gateway Node and the Gateway Node of the remote cluster.
All the traffic between the local and remote cluster is first delivered to the Gateway Node, then tunneled towards the remote gateway and finally delivered to the desired destination.

> The traffic can leave the local cluster as is in case the home and remote addressing spaces do not overlap; otherwise, the traffic crosses a properly configurated NAT in order to avoid overlapped spaces.

Reviewer: Who configures the NAT?

Author: The iptables rules configuring the NAT are managed by the routeOperator.

The TunnelEndpoint-Operator runs as deployment only on the local Gateway Node.

### Features
* GRE tunnel as VPN tunnel

### Limitations

> * Does not support other VPN solutions

Reviewer: So, it means that supports only GRE tunnels?

Author: yes, only GRE tunnels.

> * Traffic flows as it is, does not add security 

Reviewer: Should we write "Unsupported security policies" instead?

# TunnelEndpoint-Operator
## Overview

> TunnelEndpoint is the component in charge to bring up a VPN tunnel with the peering clusters.

Reviewer 2: ```suggestion
TunnelEndpoint is the component in charge of building up a VPN tunnel with the peering clusters.
```
